draft-ietf-quic-transport.md

```diff
@@ -1806,11 +1806,12 @@ support this, multiple NEW_CONNECTION_ID messages are needed.  Each
 NEW_CONNECTION_ID is marked with a sequence number.  Connection IDs MUST be used
 in the order in which they are numbered.
 
-An endpoint that to break linkability upon changing networks MUST use a
-previously unused connection ID provided by its peer.  Protection of packet
-numbers ensures that packet numbers cannot be used to correlate connections.
-Other properties of packets, such as timing and size, might be used to correlate
-activity, but no explicit correlation can be used to link activity across paths.
+Upon changing networks an endpoint MUST use a previously unused connection ID
+provided by its peer.  This eliminates the use of the connection ID for linking
+activity from the same connection on different networks.  Protection of packet
+numbers ensures that packet numbers cannot be used to correlate activity.  Other
+properties of packets, such as timing and size, might be used to correlate
+activity, but these changes ensure that the packet header cannot be used.
 
```

Comment:
"but these changes" is a bit vague - how about "but using a path unique connection ID enures ..."

Reply:
Looks good to me, but I agree with @Mikkelfj, "but these changes" is loose text and wont stand well after another change or three.